aws.yaml

```
@@ -363,9 +363,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.2.3
+  name: v9.2.4
 spec:
   state: active
+  date: 2020-04-22T12:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: chart-operator
+    version: 0.12.1
```

```suggestion
    version: 0.13.0
```

@sslavic @rossf7 should we also include latest chart-operator here? https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0130-2020-04-21

---

No time to include it. Let's do it in 9.2.5.

---

Sorry, too late for this one //cc @rossf7